common/views/components/Caption/Caption.js

```diff
@@ -1,23 +1,28 @@
 // @flow
 import {font, spacing} from '../../../utils/classnames';
-import Icon from '../Icon/Icon';
 import type {Node} from 'react';
 import PrismicHtmlBlock from '../PrismicHtmlBlock/PrismicHtmlBlock';
 import type {HTMLString} from '../../../services/prismic/types';
 
 type Props = {|
   caption: HTMLString,
-  preCaptionNode?: Node
+  preCaptionNode?: Node,
+  width?: ?number
```

Comment:
We know we'll always pass it a number, we could make it optional, not nullable?

Reply:
Actually, it's a requirement for this to be nullable.